README.md

```diff
@@ -92,9 +92,9 @@ additional resources.
 
 * [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (`v0.26.0` or
   newer) along with `podman` (`v5.3.1` or newer) or `docker` (`v27.0.1` or newer)
-* `kubectl` (`v1.31.1` or newer)
+* `kubectl` with server-side apply support
```

Comment:
```suggestion
* `kubectl` (`v1.31.1` or newer -- server-side apply support is required)
```
I'd still keep the version number, otherwise, users will have to go to the release notes to figure out what's the minimal version.